api/tests/Feature/ExperienceTest.php

```diff
@@ -66,9 +65,7 @@ public function testSkillRelationshipsWorkWithPivot(): void
             $userSkills[2]->id => ['details' => 'third skill'],
         ]);
 
-        $response = $this->actingAs($this->platformAdmin, 'api')->graphQL(
-            /** @lang GraphQL */
-            '
+        $this->actingAs($this->platformAdmin, 'api')->graphQL(<<<'GRAPHQL'
             query getUser($id: UUID!) {
```

Comment:
just curious why you changed this formatting?

Reply:
Huh, nice.  I didn't even notice that the graphql formatting wasn't working.  With this new style I get errors at least.   :+1: